WordPress/twentytwentyfive
# Services - Three-column boxed pricing table

### Testing Instructions
Create a new page.
Add the pattern.
Confirm that it looks like the designs (spacing/colors/etc.).

Closes: https://github.com/WordPress/twentytwentyfive/issues/76
[Figma](https://www.figma.com/design/dzGCSntVch4EQdVERTqyVK/Twenty-Twenty-Five?node-id=3177-1809&m=dev)
